Defs/Ammo/AmmoCategoryDefs/AmmoCategories_Grenades.xml

```diff
@@ -57,4 +57,17 @@
 		<advanced>true</advanced>
 	</CombatExtended.AmmoCategoryDef>
 
+	<CombatExtended.AmmoCategoryDef>
+		<defName>Dead</defName>
+		<label>Deadlife</label>
+		<description>Filled with a dust-like substance that explodes into a cloud and reanimates nearby corpses.</description>
+		<advanced>true</advanced>
+	</CombatExtended.AmmoCategoryDef>
+
+	<CombatExtended.AmmoCategoryDef>
```

Comment:
Flamer fuel category is inside the Advanced ammo categories file.

Reply:
Moved the category, thanks.

If we leave it as a solid, it doesn't make sense that it can be fired from regular flamethrowers (which I think is strongly preferable). Could make the recipe a mix of bioferrite + chemfuel/prometheum, with the bioferrite as essentially a crystalline suspension.